mmeval/metrics/word_accuracy.py

```diff
@@ -0,0 +1,102 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import re
+from typing import Dict, List, Sequence, Tuple, Union
+
+from mmeval.core import BaseMetric
+
+
+class WordAccuracy(BaseMetric):
+    """Calculate the word level accuracy.
+
+    Args:
+        mode (str or list[str]): Options are:
+            - 'exact': Accuracy at word level.
+            - 'ignore_case': Accuracy at word level, ignoring letter
+              case.
+            - 'ignore_case_symbol': Accuracy at word level, ignoring
+              letter case and symbol. (Default metric for academic evaluation)
+            If mode is a list, then metrics in mode will be calculated
+            separately. Defaults to 'ignore_case_symbol'.
+        valid_symbol (str): Valid characters. Defaults to
+            '[^A-Z^a-z^0-9^\u4e00-\u9fa5]'.
```

Comment:
Maybe should add some explanations on this. Is it required to be a regular expresion? What format is acceptable?